fedora-ostree-pruner/Dockerfile

```diff
@@ -7,8 +7,8 @@ ENV PYTHONUNBUFFERED=true
 # Get any latest updates since last container spin
 RUN dnf update -y
 
-# Install ostree
-RUN dnf -y install ostree
+# Install ostree/dumb-init
```

Comment:
Aside: this comment should probably be removed.

Reply:
because it's superfluous?

Reply:
Yeah, exactly.